salt/states/pcs.py

```diff
@@ -340,16 +343,20 @@ def _item_present(name, item, item_id, item_type, show='show', create='create',
         extra_args=extra_args,
         cibfile=cibfile)
 
-    log.trace('Output of pcs.item_create: {0}'.format(str(item_create)))
+    log.trace('Output of pcs.item_create: %s', item_create)
 
     if item_create['retcode'] in [0]:
-        ret['comment'] += 'Created {0} {1} ({2})\n'.format(str(item), str(item_id), str(item_type))
-        ret['changes'].update({item_id: {'old': '', 'new': str(item_id)}})
+        ret['comment'] += 'Created {0} {1} ({2})\n'.format(
+            six.text_type(item), six.text_type(item_id), six.text_type(item_type)
+        )
+        ret['changes'].update({item_id: {'old': '', 'new': six.text_type(item_id)}})
     else:
         ret['result'] = False
-        ret['comment'] += 'Failed to create {0} {1} ({2})\n'.format(str(item), str(item_id), str(item_type))
+        ret['comment'] += 'Failed to create {0} {1} ({2})\n'.format(
+            six.text_type(item), six.text_type(item_id), six.text_type(item_type)
```

Comment:
Same as above, we can remove all instances of `six.text_type()` on this line and just pass the variables.